src/client.rs

```diff
@@ -153,19 +153,20 @@ fn from_process(process: &str) -> Option<Vec<String>> {
 
 #[cfg(target_family = "windows")]
 fn from_process(process: &str) -> Option<Vec<String>> {
+    use std::os::windows::process::CommandExt;
+    const CREATE_NO_WINDOW: u32 = 0x08000000;
+
     let wmic = process::Command::new("WMIC")
         .args(["path", "win32_process", "get", "Caption,Commandline"])
-        .stdout(process::Stdio::piped())
-        .spawn()
+        .creation_flags(CREATE_NO_WINDOW)
+        .output()
         .ok()?;
 
-    let process_exe = format!("{}.exe", process);
+    let stdout = String::from_utf8(wmic.stdout).ok()?;
 
-    let mut findstr = process::Command::new("findstr");
-    findstr.args(["/R", &process_exe]).stdin(wmic.stdout?);
+    let process_exe = format!("{}.exe", process);
 
-    let output = String::from_utf8(findstr.output().ok()?.stdout).ok()?;
-    let lines = output.lines();
+    let lines = stdout.lines().filter(|x| x.contains(&process_exe));
```

Comment:
could you just add in the stuff below as well?

```rust
let lines: Vec<String> = stdout.lines().filter(|x| x.contains(&process_exe)).filter(|x| x.contains("--app-port") && x.contains("--remoting-auth-token")).map(String::from).collect();
```

Reply:
yeah, i always forget to cleanup the code